### ISSUE TITLE
Release v0.108.0 breakes yandex_storage_bucket

### ISSUE THREAD

**Issue by darzanebor** [yandex-cloud/terraform-provider-yandex#414]
Release v0.108.0 breakes yandex_storage_bucket
```
Stack trace from the terraform-provider-yandex_v0.108.0 plugin:

panic: interface conversion: interface {} is []interface {}, not []map[string]interface {}

goroutine 145 [running]:
github.com/yandex-cloud/terraform-provider-yandex/yandex.resourceYandexStorageBucketStateUpgradeV0({0x0?, 0x0?}, 0xc0014f9d10, {0x0?, 0x0?})
        github.com/yandex-cloud/terraform-provider-yandex/yandex/resource_yandex_storage_bucket_migrate.go:499 +0x285
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).upgradeJSONState(0xc000169188, {0x26ff6c0, 0xc0014f84b0}, 0xd?, 0x23c9872?, 0x14?)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.28.0/helper/schema/grpc_provider.go:452 +0x7d
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).UpgradeResourceState(0xc000169188, {0x26ff6c0?, 0xc0014f8390?}, 0xc0006fd0a0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.28.0/helper/schema/grpc_provider.go:328 +0x437
github.com/hashicorp/terraform-plugin-mux/tf5to6server.v5tov6Server.UpgradeResourceState({{0x2709e28?, 0xc000169188?}}, {0x26ff6c0?, 0xc0014f8390?}, 0xc0006fd040?)
        github.com/hashicorp/terraform-plugin-mux@v0.11.2/tf5to6server/tf5to6server.go:130 +0x147
github.com/hashicorp/terraform-plugin-mux/tf6muxserver.muxServer.UpgradeResourceState({0xc000dfe7b0, 0xc0010b5b80, 0xc000dfe7e0, 0xc000dfe810, {0xc00085e880, 0x2, 0x2}}, {0x26ff6c0?, 0xc0014f8060?}, 0xc0006fd040)
        github.com/hashicorp/terraform-plugin-mux@v0.11.2/tf6muxserver/mux_server_UpgradeResourceState.go:30 +0x139
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).UpgradeResourceState(0xc00028f220, {0x26ff6c0?, 0xc0014e98f0?}, 0xc00145e730)
        github.com/hashicorp/terraform-plugin-go@v0.18.0/tfprotov6/tf6server/server.go:717 +0x270
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_UpgradeResourceState_Handler({0x22d1360?, 0xc00028f220}, {0x26ff6c0, 0xc0014e98f0}, 0xc0014d0f80, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.18.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:350 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000ad3200, {0x26ff6c0, 0xc0014e9860}, {0x27085e0, 0xc00056c000}, 0xc0014ee480, 0xc000ea8690, 0x3ff2840, 0x0)
        google.golang.org/grpc@v1.61.0/server.go:1385 +0xe49
google.golang.org/grpc.(*Server).handleStream(0xc000ad3200, {0x27085e0, 0xc00056c000}, 0xc0014ee480)
        google.golang.org/grpc@v1.61.0/server.go:1796 +0x1031
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.61.0/server.go:1029 +0x91
created by google.golang.org/grpc.(*Server).serveStreams.func2
        google.golang.org/grpc@v1.61.0/server.go:1040 +0x145

Error: The terraform-provider-yandex_v0.108.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**Comment by darzanebor** [comment]
Issue in migration of state with resource_yandex_storage_bucket_migrate.go and not in creation of resources with same code and new provider as I assume

**Comment by PeppaTheC** [comment]
Hi, the issue must be fixed in release [`0.108.1`](https://github.com/yandex-cloud/terraform-provider-yandex/releases/tag/v0.108.1).

Please, upgrade provider version.
